akka-stream/src/main/scala/akka/stream/SystemMaterializer.scala

```diff
@@ -26,5 +26,5 @@ object SystemMaterializer extends ExtensionId[SystemMaterializer] with Extension
 }
 
 final class SystemMaterializer(system: ExtendedActorSystem) extends Extension {
-  val materializer = ActorMaterializer(ActorMaterializerSettings(system), "flow")(system)
+  val materializer: Materializer = ActorMaterializer(ActorMaterializerSettings(system), "flow")(system)
```

Comment:
I don't see that it was internal. It's public `val`. It must anyway be fixed, and hopefully not too many libraries has started using 2.5.29 when we release 2.5.30.

We should also fix master to use explicit type on this field

Reply:
Oh, ouch.

Reply:
In master it is already inferred to the right type because of previous explicit type (but we should add the type nonetheless)

Reply:
(agreed, added, #28545)

Reply:
> I don't see that it was internal. It's public `val`. It must anyway be fixed, and hopefully not too many libraries has started using 2.5.29 when we release 2.5.30.

Do we have a 2.5.30 release planned any time soon? Otherwise, that statement might not be true any more.